File: Studio Classico/src/main/java/es/uco/iw/model/InstrumentCount.java
package es.uco.iw.model;

public class InstrumentCount {
	private int count;
	private Instrument instrument;
	
	public InstrumentCount(int count, Instrument instrument) {
		setCount(count);
		setInstrument(instrument);
	}
	
	public int getCount() {
		return count;
	}
	
	public Instrument getInstrument() {
		return instrument;
	}
	
	public void setCount(int count) {
		this.count = count;
	}

	public void setInstrument(Instrument instrument) {
		this.instrument = instrument;
	}
}

File: Studio-Classico/app/Models/InstrumentCount.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class InstrumentCount extends Model
{
    use HasFactory;

    protected $table = "pieces_instruments";

    protected $fillable = [
        'count',
        'instrument_id'
    ];

    public function piece(){
        return $this->belongsTo(Piece::class);
    }

    public function instrument(){
        return $this->belongsTo(Instrument::class);
    }

}